doc/specs/#117-Windows-Store-Support.md

```diff
@@ -0,0 +1,144 @@
+
+#### GA Only
+We are generic and only pull down the latest GA version of the app.  
+
+## UI/UX Design
+Each Store app in the repo will be interacted with by the client like any other app by the Windows Package Manager.
+
+```
+PS C:\> winget search appinstaller
+Name                    Id                                Version
+----------------------------------------------------------------------
+AppInstallerFileBuilder Microsoft.AppInstallerFileBuilder 1.2020.211.0
+```
+And Install
+### Store App installation experience
+Installing the store app will provide the MSIX installation experience.  The Windows Store App will stream install.
+
+![](images\117-Download.png)
+
+
+## Manifests
+Developers will be able to submit their app by providing meta data that includes the store ID or URL to the store app.
+
+### Duplicate app entries
+With the addition of the store, we will have an increase in duplicate entries.  If we have 2 manifests with the same application, we should try to eliminate the duplicate entries by default.  Otherwise it will make it difficult to limit the selection to the one file.
+
+#### Duplicate manifests
+Because we have manifests that are based on publisher and appname, there should not be a case where we have multiple manifests with different installers.  If we do, there is something wrong with the metadata.  
+ 
+#### Duplicate entries, one manifest
+If a manifest file has multiple formats, MSI, EXE, MSIX.  Unless specified by the user, default will be to display and install will be the  MSIX format.
+
+
+<b>  Example: <appname>  </b>  
+```
+...  
+InstallerType: MSI  
+...  
+InstallerType: MSIX  
+```
+When the user specifies install by default the client will choose to install the version of the app that is flagged as default in the manifest.  If no default is specified, then the app will default to the store as the default app.
+
+See [Install-Order-Spec.md](.\Install-Order-Spec.md) for more details.
+
+
+## Submission Wizard
+Much of the metadata to create a manifest file is available from the homepage.  A tool will be created to pull the data and create a manifest file for adding to the Windows Store.
+
+
+## Capabilities
+
+### Accessibility
+
+N/A - as the store apps should be no different than other apps
+
+### Security
+
+We need to evaluate access to the API and ensure that for purchase apps are not exposed.
+
+### Reliability
+
+N/A - as the store apps should be no different than other apps.
+
+### Compatibility
+
+Changes to support the store will be in addition.  They should not impact compatibility.  However the metadata changes to the manifest spec will mean newer apps may have additional entries to add.
+
+### Performance, Power, and Efficiency
+
+## Potential Issues
+
+
+## Future considerations
+
+The wizard to ease metadata transfer.
+
+## Resources
+
+[comment]: # Be sure to add links to references, resources, footnotes, etc.
+
+## History
```

Comment:
Git keeps a history of all revisions to a document, this probably isn't necessary